Cloudshare fixed (#10682)
demisto/content: Packs/CloudShare/ReleaseNotes/1_0_2.md

#### Integrations
##### CloudShare (Beta)

> - Fixed an issue where the ***cloudshare-delete-student*** snd the ***cloudshare-get-student*** command was incorrectly handled in the code.

Reviewer: ```suggestion
- Fixed an issue where the ***cloudshare-delete-student*** and the ***cloudshare-get-student*** commands were incorrectly handled in the code.
```
